### PR TITLE
[W5.7][W12-1] Bobbie Ramadhani Soedirgo

### PR DESCRIPTION
**Pull request by soedirgo**

### CODE REVIEW

**Comment by Swiftkage** [comment]
Very interesting feature. Good work.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+/**
```
Comment (asiddharth) [inline src/seedu/addressbook/commands/SortCommand.java:11]: Good job on adding the header comment

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * Sorts all persons in the address book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
```
Comment (asiddharth) [inline src/seedu/addressbook/commands/SortCommand.java:17]: Extra blank line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

**Review by asiddharth — APPROVED** [review]
Kudos! The code looks clean, has tests, is documented. great job @dan-delion !

**Pull request closed without merging**